docs/source/release_notes/release-notes.rst

```diff
@@ -69,7 +88,8 @@ develop
            (`Pull Request <https://github.com/palantir/atlasdb/pull/2238>`__)
 
     *    - |devbreak|
-         - Throw an error during schema code generation stage if table length exceeds KVS limits.
+         - AtlasDB now throws an error during schema code generation stage if table length exceeds KVS limits.
```

Comment:
This devbreak should be moved above the 2238 devbreak, since it's more likely to actually affect someone.